src/core/Akka.Remote/Transport/Helios/HeliosTransport.cs

```diff
@@ -154,13 +161,18 @@ private int ComputeWps(Config config)
     /// </summary>
     abstract class HeliosTransport : Transport
     {
+
+        private readonly IEventLoopGroup _serverEventLoopGroup;
```

Comment:
The private threadpools managed by Helios, which are cleaned up when the transport is terminated.